packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -853,6 +857,20 @@ export default defineComponent({
 					return;
 				}
 
+				if (
+					parameterData.type &&
+					SHOULD_CLEAR_NODE_OUTPUTS[nodeType.name] &&
```

Comment:
Eslint warning here: `Prefer using an optional chain expression instead, as it's more concise and easier to read.eslint[@typescript-eslint/prefer-optional-chain](https://typescript-eslint.io/rules/prefer-optional-chain)`